### PR TITLE
Add capistrano

### PR DESCRIPTION
**Pull request by gabrielpoca**

### CODE REVIEW

**Comment by gabrielpoca** [comment]
@naps62 please let me know how it looks.

File: config/deploy.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+set :application, 'apprenticeship.subvisual.co'
+set :repo_url, 'git@github.com:groupbuddies/apprenticeship.git'
+set :stage, :production
+set :branch, :master
+
```
Comment (naps62) [inline config/deploy.rb:6]: please add `set :linked_dirs, %w(log)`

i've configured nginx to log everything inside /apprenticeship to that directory

Comment (gabrielpoca) [inline config/deploy.rb:6]: Done

**Comment by naps62** [comment]
:+1:

**Pull request merged**